store/mongo/migrations.go

```diff
@@ -46,26 +47,37 @@ func Migrate(ctx context.Context,
 		return errors.Wrap(err, "failed to parse service version")
 	}
 
-	m := migrate.SimpleMigrator{
-		Client:      client,
-		Db:          db,
-		Automigrate: automigrate,
+	dbNames, err := client.ListDatabaseNames(ctx, bson.D{
+		{Key: "name", Value: bson.D{
+			{Key: "$regex", Value: "^" + DbName + "-"},
+		}},
+	})
+	if err != nil {
+		return err
 	}
 
-	migrations := []migrate.Migration{
-		&migration_1_0_0{
-			client: client,
-			db:     db,
-		},
-		&migration_2_0_0{
-			client: client,
-			db:     db,
-		},
-	}
+	for _, dbName := range append([]string{DbName}, dbNames...) {
+		m := migrate.SimpleMigrator{
+			Client:      client,
+			Db:          dbName,
+			Automigrate: automigrate,
+		}
 
-	err = m.Apply(ctx, *ver, migrations)
-	if err != nil {
-		return errors.Wrap(err, "failed to apply migrations")
+		migrations := []migrate.Migration{
+			&migration_1_0_0{
+				client: client,
+				db:     dbName,
+			},
+			&migration_2_0_1{
+				client: client,
+				db:     dbName,
+			},
+			// NOTE: Future migrations need only be applied to DbName
```

Comment:
but not the `dbName` ;-) I would somehow underline that. or add another comment after the loop.